components/server/src/workspace/workspace-starter.ts

```diff
@@ -657,6 +658,12 @@ export class WorkspaceStarter {
                 });
             }
         } catch (err) {
+            if (!(err instanceof StartInstanceError)) {
```

Comment:
we looked at logs, there was grpc unavailable should be retried, but other were good

for unavailable, maybe we can also improving messaging for a user if it helps

Reply:
I added an exception to make these cases (gRPC errors UNAVAILABLE and ALREADY_EXISTS) a transient failure: We log the failure, but don't fail the workspace start:
 - UNAVAILABLE should be temporary, and will be fixed by a follow-up attempt
 - ALREADY_EXISTS will be fixed by receiving a first any update from ws-manager